### PR TITLE
Update action to node 20

### PR DESCRIPTION
**Pull request by anttiharju**
**What**

Title + minor clean up while I'm here.

**Why**

Produces a ton of warnings about the node 16 deprecation for example in the starter repo (and many others).

### CODE REVIEW

**Comment by anttiharju** [comment]
Seemingly in this repo reviews are not required, should fix that. (will ask on monday)

File: package.json [hunk 1]
```diff
@@ -28,22 +28,22 @@
...
     "jest-circus": "29.7.0",
     "js-yaml": "4.1.0",
-    "prettier": "2.8.8",
+    "prettier": "3.2.5",
     "ts-jest": "29.1.2",
-    "typescript": "4.9.5"
+    "typescript": "5.4.4"
   }
-}
+}
```
Comment (JohnDing1995) [inline package.json:49]: Add a line end of the file

**Review by JohnDing1995 — APPROVED** [review]
otherwise looks good

File: package.json (hunk 1 above)
Comment (anttiharju) [reply package.json]: Nice catch!

**Pull request merged**